src/crimson/osd/pg_backend.cc

```diff
@@ -215,6 +216,100 @@ PGBackend::read(const object_info_t& oi,
     });
 }
 
+namespace {
+
+  template<class CSum>
+  PGBackend::checksum_errorator::future<>
```

Comment:
How about formatting like below?

```suggestion
  PGBackend::checksum_errorator::future<> do_checksum(
    size_t chunk_size,
    const ceph::bufferlist& buf,
    ceph::bufferlist& result)
  {
```

Reply:
i understand you want to be consistent on the formatting in this source file. but IMHO, the existing layout is easier on eyes. for scanning the function signature, the read's focus would be like

```
  |
  |
  \--------\
           |
           |
           |
           |
 ```

with the formatting you recommend, he/she have to
```
  |
  |
  \--------\
         /
       /
     / 
   /
  |
  |
  |
  |
  |
```

Reply:
I'm buying this. Later we might refactor the file.